router/image/templates/nginx.conf

```diff
@@ -157,6 +158,12 @@ http {
 
             proxy_next_upstream         error timeout http_502 http_503 http_504;
 
+            {{ if ne $enforceHttps "false" }}
```

Comment:
@bacongobbler just in case the $enforceHttps returns a string that's why the comparison is required. 
Unless the actual value is not important and only the existence of the key indicates the use of a "feature".